modules/apim_gateway/manifests/startserver.pp

```diff
@@ -18,8 +18,14 @@
 # Starts the server as a service in the final stage.
 class apim_gateway::startserver inherits apim_gateway::params {
 
+  exec { 'daemon-reload':
+    command => "systemctl daemon-reload",
+    path    => "/bin/",
+  }
+
   service { $service_name:
-    ensure => running,
     enable => true,
+    ensure => running,
+    subscribe => File["binary"],
```

Comment:
Format the code

Reply:
Resolved https://github.com/wso2/puppet-apim/pull/86/commits/5dd25054bc378c8b256e4571b433b51eecc8d807